sorc/checkout.sh

```diff
@@ -9,10 +9,6 @@ do
    echo "Received -o flag for optional checkout of GTG, will check out GTG with EMC_post"
    checkout_gtg="YES"
    ;;
-  c)
```

Comment:
Same issue as before. coupled-crow uses the c switch to checkout and build the coupled model.

Reply:
Maybe we can get rid of the -c option in feature/coupled-crow too?  After all you can build the coupled exe and still run atmosphere only.

Reply:
We could do that. My only concern is if/when we merge coupled-crow back to develop we're now locked into using the same ufs commit for both. This may not really be an issue though.

Reply:
That sounds like a positive not a negative to me.   Someone can always manually update the version to be checked out.

Reply:
Having the two options is somewhat of a hold over from the ufs-s2s-model versus ufs-weather-model.  Now the difference is that the previous s2s checks out into a "ufs-coupled.fd" instead of the fv3gfs folder.

Reply:
Are we ok with me removing the -c option in develop and it's impact on feature/coupled-crow?

Reply:
I am